### ISSUE TITLE
Grid Docs Feedback

### ISSUE THREAD

**Issue by gauntface** [google/material-design-lite#1360]
Just tried to use the Grid but had trouble understanding that the grid size changing depending on the viewport size.

What would have helped is if this was explained more prominently on the doc.

Maybe even basic diagram of phone, tablet and desktop with number of single blocks supported.

**Comment by surma** [comment]
The [doc for grid](http://www.getmdl.io/components/index.html#layout-section/grid) mentions all of this.

I feel like this is an artifact of our text-based documentation not getting the attention it deserves in general. Maybe we should rethink the way our components page prioritizes/emphasizes the docs?
@sgomes @nicolasgarnier WDYT?

**Comment by sgomes** [comment]
Yup, @gauntface was commenting that we should have some text first, explaining the most important grid concepts. The reference is a bit of a wall of text, sadly.

We might want to consider a rework of the docs pages in the future, with an intro followed by demos and then reference. Or perhaps combining demos and intro into a single, customised section.

We may also want to revise some of the text in the reference, as it's very boilerplate-y, and could probably be simplified (or in some cases removed).
